pilot/pkg/config/kube/crdclient/types.gen.go

```diff
@@ -205,6 +206,103 @@ func update(ic versionedclient.Interface, sc serviceapisclient.Interface, cfg co
 	}
 }
 
+func updateStatus(ic versionedclient.Interface, sc serviceapisclient.Interface, cfg config.Config, objMeta metav1.ObjectMeta) (metav1.Object, error) {
+	switch cfg.GroupVersionKind {
+	case collections.IstioNetworkingV1Alpha3Destinationrules.Resource().GroupVersionKind():
+		return ic.NetworkingV1alpha3().DestinationRules(cfg.Namespace).UpdateStatus(context.TODO(), &clientnetworkingv1alpha3.DestinationRule{
+			ObjectMeta: objMeta,
+			Spec:       *(cfg.Spec.(*networkingv1alpha3.DestinationRule)),
+			Status:     *(cfg.Status.(*metav1alpha1.IstioStatus)),
```

Comment:
I think status may be `nil` here since we don't check it in UpdateStatus. Spec doesn't have the same issue since we do check it explicitly there.